packages/sdk/src/wallet/index.ts

```diff
@@ -27,4 +35,41 @@ export class Wallet implements WalletInterface {
   async getBalance(): Promise<bigint> {
     return 0n // TODO: placeholder
   }
+
+  /**
+   * Lend assets to a lending market
+   * @description Lends assets using the configured lending provider
+   * @param asset - Asset token address to lend
+   * @param amount - Amount to lend (in wei)
+   * @param marketId - Optional specific market ID
+   * @param options - Optional lending configuration
+   * @returns Promise resolving to lending transaction details
+   * @throws Error if no lending provider is configured
+   */
+  async lend(
+    asset: Address,
```

Comment:
for the sdk, do we expect the consumer to put in the asset address or would they just provide the symbol and then the sdk is responsible for finding the best place to lend?